datahub-web-react/src/app/ingest/source/builder/RecipeForm/SecretField/SecretField.tsx

```diff
@@ -34,9 +58,36 @@ interface SecretFieldProps {
     refetchSecrets: () => void;
 }
 
+function SecretFieldTooltip({ tooltipLabel }: { tooltipLabel?: string | ReactNode }) {
+    return (
+        <div>
+            {tooltipLabel}
+            <hr />
```

Comment:
can you add a check that the tooltipLabel exists first before adding this horizontal line? so you could do:
```
{tooltipLabel && (<>{tooltipLabel}<hr /></>)}
```